update restified endpoints docs (#972)
hasura/ddn-docs: docs/plugins/restified-endpoints/how-to.mdx

RESTified GraphQL endpoints to DDN. This can be used to add custom REST endpoints to DDN that would execute specified
GraphQL query on the DDN GraphQL API and return the response.

This guide provides a step-by-step guide on how to configure and deploy the RESTified Endpoints Plugin for Hasura DDN
using either of the two methods:

> 1. [Using docker image](#using-docker-image)

Reviewer: ```suggestion
1. [Using a Docker image](#using-docker-image)
```

2. [Using Cloudflare Wrangler](#using-cloudflare-wrangler)

> ## Using Docker Image

Reviewer: ```suggestion
## Using a Docker Image
```

In this example, we're going to use a Docker image to configure the RESTified Endpoints Plugin for Hasura DDN. You can

> use similar configurations to deploy the plugin using kubernetes or any other container orchestration tool.

Reviewer: ```suggestion
use similar configurations to deploy the plugin using Kubernetes or any other container orchestration tool.
```

:::

### Step 3. Run everything

At this point, you can run the following command to build and start local development:

```bash
ddn supergraph build local
ddn run docker-start
```

You can now test the plugin by making a request to the RESTified GraphQL endpoints defined in the plugin's
configuration.

If you want to make changes to the plugin configuration, you can update the `configuration.json` file and restart the
docker services.

### Step 4. Deploy the plugin

The plugin can be deployed as a regular HTTP service using any container orchestration tool. When deployed, make sure to
set the `OTEL_EXPORTER_OTLP_ENDPOINT`, `OTEL_EXPORTER_PAT`, `GRAPHQL_SERVER_URL` and `HASURA_DDN_PLUGIN_CONFIG_PATH` to
appropriate values in `.cloud.env`. Note that the plugin must be visible from your Hasura deployment: if you are hosting
for the Hasura Cloud, the plugin must be publicly visible.

:::info Your plugin must be reachable

> Since the rate limit plugin is a pre-parse plugin, it must be reachable from the Hasura deployment. If you are hosting

Reviewer 2: ```suggestion
Since the RESTified endpoints plugin is a pre-parse plugin, it must be reachable from the Hasura deployment. If you are hosting
```

Author: My bad.
